test/e2e/healthcheck_run_test.go

```diff
@@ -263,6 +263,20 @@ var _ = Describe("Podman healthcheck run", func() {
 		Expect(ps.OutputToString()).To(ContainSubstring("hc"))
 	})
 
+	It("hc logs do not include exec events", func() {
+		session := podmanTest.Podman([]string{"run", "-dt", "--name", "hc", "--health-cmd", "true", "--health-interval", "5s", "alpine", "sleep", "60"})
+		session.WaitWithDefaultTimeout()
+		Expect(session).Should(ExitCleanly())
```

Comment:
You might want to `defer (podman stop hc)`. As a general rule it's a good idea to do your own cleanup, not leave it for teardown to have to do.

Reply:
done, thanks!

Reply:
I'm sorry, I used unwarranted shortcuts in my hasty comment. Plain `stop` is a very bad thing to do on a container running `sleep`: it will waste ten seconds (and those seconds add up over the course of thousands of tests). Also, there's no `Wait` or status check.

I've looked in the code base for an example to emulate, and find none, so it looks like `defer` is not the right approach. And other tests in this module don't bother cleaning up, so I retract my request with apologies. Just remove the `defer`, let teardown take care of cleanup.

Reply:
removed the line, thanks